arcgis-ios-sdk-samples/Display information/Filter by time extent/FilterByTimeExtent.storyboard

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="19455" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="Y6W-OH-hqX">
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="19454"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="System colors in document resources" minToolsVersion="11.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Filter By Time Extent View Controller-->
+        <scene sceneID="s0d-6b-0kx">
+            <objects>
+                <viewController id="Y6W-OH-hqX" customClass="FilterByTimeExtentViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="5EZ-qb-Rvc">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="4xT-6d-gMC" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="0.0" width="414" height="862"/>
+                            </view>
+                        </subviews>
+                        <viewLayoutGuide key="safeArea" id="vDu-zF-Fre"/>
+                        <color key="backgroundColor" systemColor="tertiarySystemBackgroundColor"/>
+                        <constraints>
+                            <constraint firstItem="4xT-6d-gMC" firstAttribute="trailing" secondItem="vDu-zF-Fre" secondAttribute="trailing" id="6aX-97-lce"/>
+                            <constraint firstItem="vDu-zF-Fre" firstAttribute="bottom" secondItem="4xT-6d-gMC" secondAttribute="bottom" id="MQC-gN-tDP"/>
+                            <constraint firstItem="4xT-6d-gMC" firstAttribute="leading" secondItem="vDu-zF-Fre" secondAttribute="leading" id="QwE-1a-raV"/>
```

Comment:
Please snap the map view to superview, instead of safe area. And update screenshot 📸 

![landscape](https://user-images.githubusercontent.com/9660181/142504926-51d39c9a-c087-4d02-9307-904d1b2498e6.png)